src/Sarif/Autogenerated/ReportingDescriptor.cs

```diff
@@ -163,9 +164,9 @@ public ReportingDescriptor()
         /// <param name="properties">
         /// An initialization value for the <see cref="P:Properties" /> property.
         /// </param>
-        public ReportingDescriptor(string id, string guid, string name, IEnumerable<string> deprecatedIds, MultiformatMessageString shortDescription, MultiformatMessageString fullDescription, IDictionary<string, MultiformatMessageString> messageStrings, ReportingConfiguration defaultConfiguration, Uri helpUri, Message help, IEnumerable<ReportingDescriptorReference> taxonomyReferences, IEnumerable<ReportingDescriptorReference> optionalTaxonomyReferences, IDictionary<string, SerializedPropertyInfo> properties)
```

Comment:
>string name, IEnumerable<string> deprecatedIds, [](start = 59, length = 47)

this was before i learnt i can actually generate not-yet-auto-gen files ;) this PR corrects the sequence to follow wat the auto-gen logic would do.